graph2tac/tfgnn/dataset.py

```diff
@@ -516,6 +196,11 @@ def _make_bare_graph_tensor(node_labels: tf.Tensor,
         @param edge_labels: tf.Tensor of edge labels (dtype=tf.int64)
         @return: a GraphTensor object that is compatible with the `bare_graph_spec` in `graph_schema.py`
         """
+        node_labels = tf.convert_to_tensor(graph.nodes, dtype = tf.int64)
+        edge_labels = tf.convert_to_tensor(graph.edge_labels, dtype = tf.int64)
+        sources = tf.convert_to_tensor(graph.edges[:,0], dtype = tf.int32)
+        targets = tf.convert_to_tensor(graph.edges[:,1], dtype = tf.int32)
+
```

Comment:
We shouldn't need these conversions if this is compiled and the TF types are specified in the spec.

Reply:
And elsewhere in this file.

Reply:
on the other hand, I think it is good to have a code which works even outside tensorflow. Otherwise, I was getting very incomprehensible error messages.